### PR TITLE
Allow to configure the interval in minutes to fetch new email

### PR DESCRIPTION
**Pull request by kalmbach**
Fixes [Add a setting to allow the user to configure the interval to fetch new email](https://github.com/elementary/switchboard-plug-onlineaccounts/issues/223)
Fixes [Configurable refresh period](https://github.com/elementary/mail/issues/752)

Captura de pantalla de 2022-01-08 12-23-07

### CODE REVIEW

File: po/online-accounts-plug.pot [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: online-accounts-plug\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-12-13 22:24+0000\n"
+"POT-Creation-Date: 2022-01-08 11:17-0300\n"
```
Comment (meisenzahl) [inline po/online-accounts-plug.pot:11]: Please reset the changes to this file. The translation files are handled by [Weblate](https://l10n.elementary.io/projects/switchboard/switchboard-plug-onlineaccounts/).

File: src/Dialogs/Imap/ImapDialog.vala [hunk 2]
```diff
@@ -68,6 +69,10 @@ public class OnlineAccounts.ImapDialog : Hdy.Window {
             value = 993
         };
 
+        imap_refresh_interval_spin = new Gtk.SpinButton.with_range(1, 60, 1) {
```
Comment (meisenzahl) [inline src/Dialogs/Imap/ImapDialog.vala:72]:
```suggestion
        imap_refresh_interval_spin = new Gtk.SpinButton.with_range (1, 60, 1) {
```

We insert white space before opening parentheses or braces.

Reference
- https://docs.elementary.io/develop/writing-apps/code-style#whitespace

**Review by meisenzahl — CHANGES_REQUESTED** [review]
The code looks good so far, just a bit nitpicking 🤓

**Comment by meisenzahl** [comment]
@kalmbach thanks for implementing my comments so quickly. Is it also possible to change the interval for an already configured account in edit mode?

Bildschirmfoto von 2022-01-08 17 25 32

**Comment by kalmbach** [comment]
Thanks for the review @meisenzahl !!

**Comment by kalmbach** [comment]
> @kalmbach thanks for implementing my comments so quickly. Is it also possible to change the interval for an already configured account in edit mode?

@meisenzahl yarp! It is.

**Comment by meisenzahl** [comment]
@kalmbach nice! Let's wait for some input of @elementary/ux.

**Review by meisenzahl — APPROVED** [review]

**Review by danrabbit — APPROVED** [review]
Nice work!

**Pull request merged**